src/_includes/docs/swift-package-manager/migrate-ios-project-manually.md

```diff
@@ -0,0 +1,76 @@
+Once you enable the Swift Package Manager feature, the Flutter CLI will try
+to migrate your project to add Swift Package Manager integration. This makes
+your project download the Swift packages that your Flutter plugins depend on.
+
+However, the Flutter CLI may be unable to migrate your project if you've
+modified it significantly.
+
+Please [file a bug][] before migrating manually to help the Flutter team improve
+the automatic migration.
+Please include the error message you received and consider including a copy of
+the of the following files in your bug report:
+
+* `ios/Runner.xcodeproj/project.pbxproj`
+* `ios/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme`
+   (or the xcsheme for the flavor used)
+
+### Step 1: Add FlutterGeneratedPluginSwiftPackage Package Dependency {:.no_toc}
+
+1. Open your app (`ios/Runner.xcworkspace`) in Xcode.
+2. Navigate to Package Dependencies for the project.
+
+   {% render docs/captioned-image.liquid,
+   image:"development/packages-and-plugins/swift-package-manager/package-dependencies.png",
+   caption:"The project's package dependencies" %}
+
+3. Click the `+` button.
+4. Click the `Add Local...` button on the bottom of the dialog that opens.
+5. Navigate to `ios/Flutter/ephemeral/Packages/FlutterGeneratedPluginSwiftPackage`
+   and click the `Add Package` button.
+6. Ensure it is added to the `Runner` target and click the `Add Package` button.
+ 
+   {% render docs/captioned-image.liquid,
+   image:"development/packages-and-plugins/swift-package-manager/choose-package-products.png",
+   caption:"Ensure the package is added to the `Runner` target" %}
+
+7. Ensure `FlutterGeneratedPluginSwiftPackage` was added to Frameworks,
+   Libraries, and Embedded Content.
+
+   {% render docs/captioned-image.liquid,
+   image:"development/packages-and-plugins/swift-package-manager/add-generated-framework.png",
+   caption:"Ensure `FlutterGeneratedPluginSwiftPackage` was added to Frameworks, Libraries, and Embedded Content" %}
+
+### Step 2: Add Run Prepare Flutter Framework Script Pre-Action {:.no_toc}
+
+**The following must be completed for each flavor.**
+
+1. Next, select Product > Scheme > Edit Scheme.
+2. Click the `>` next to "Build" in the left side bar.
+3. Select Pre-actions.
+4. Click the `+` button and select `New Run Script Action` from the menu.
+5. Click the "Run Script" title and change to
+   `Run Prepare Flutter Framework Script`.
+6. Change the "Provide build settings from" to the app.
+7. Input the following in the text box:
+
+   ```sh
+   /bin/sh "$FLUTTER_ROOT/packages/flutter_tools/bin/xcode_backend.sh" prepare
+   ```
+
+   {% render docs/captioned-image.liquid,
+   image:"development/packages-and-plugins/swift-package-manager/add-flutter-pre-action.png",
+   caption:"Add `Run Prepare Flutter Framework Script` build pre-action" %}
+
+### Step 3: Run app {:.no_toc}
+
+1. Run the app in Xcode and ensure `FlutterGeneratedPluginSwiftPackage` is a
```

Comment:
```suggestion
1. Run the app in Xcode and ensure that **FlutterGeneratedPluginSwiftPackage** is a
```

Reply:
`FlutterGeneratedPluginSwiftPackage` is a Swift package identifier that is displayed in something that looks like menu text... I believe backticks are preferable here, but let me know if I'm incorrect!